views/confirmation.html

```diff
@@ -16,9 +16,10 @@
     <div class="govuk-grid-column-two-thirds">
 
       <div class="govuk-panel govuk-panel--confirmation">
-        <h1 class="govuk-panel__title">Application submitted</h1>
+        <h1 class="govuk-panel__title">{{ "Update statement submitted" if isUpdate else "Application submitted" }}</h1>
```

Comment:
I _think_ you can use your config title here like `OE_CONFIGS.UPDATE_CONFIRMATION_PAGE_TITLE` instead of the hard-coded string

Reply:
Ah nice! Sorry about missing that.